docs/guide/http_cache.md

```diff
@@ -338,22 +338,14 @@ ezpublish:
 
     ### Setting Time-To-Live value for Page blocks
 
-    Page blocks are rendered using Edge Site Include which means you can set different TTL values for each Page block type.
-    The TTL setting is available in the configuration under a `ttl` key. The value has to be set in seconds:
-
-    ``` yaml
-    ez_systems_landing_page_field_type:
-        blocks:
-            block_type:
-                ttl: 600
-                views:
-                    (...)
-    ```
-
-    `block_type` should be replaced with the actual block name, e.g. `embed`, `collection`, `schedule`, etc.
-    In the example above `block_type` will be cached for 10 minutes.
-
-    By default blocks are not cached (TTL = 0) for backwards compatibility reasons.
+    Block cache by default respects `$content.ttl_cache$` and `$content.default_ttl$` settings.
+    However, if the given block value has a since / till date,
+    this will be taken into account for the TTL calculation for the block and also for the whole page.
+
+    To overload this behavior, listen to [BlockResponseEvents::BLOCK_RESPONSE](extending_page/#block-render-response).
+    For instance, to disable cache for a custom block you can set a priority of 100,
```

Comment:
Maybe I'm wrong, but besides suggestion, I'm wondering whether the priority should be `-100`, perhaps? AFAIR the `\EzSystems\EzPlatformPageFieldType\Event\Subscriber\BlockVisibilityTtlHeadersResponseSubscriber` has a priority of `-100`, so if the developer's own listener has priority of `100` and sets different cache settings for the given response, these will be overridden afterward by the PB listeners, right?

Reply:
I might have messed up the order. But the idea here was to run the listener before eZ's if you want to disable cache, and after (ref -200 below) if just want to adjust the headers that gets set by default

Reply:
@kmadejski WDYT? Or should we get someone else to review also maybe to see if this can be simplified?

Reply:
As for me, the current description is totally fine, I didn't find it too complex. The only doubt is about these priorities. I briefly checked it and it seems that indeed the priority has to be at least `-200` to be able to modify `s-maxage`/`maxage`, I believe it's the same for `public`/`private`.

Reply:
Better?